person/tests.py

```diff
@@ -0,0 +1,876 @@
+import datetime
+from django.test import TestCase
+from person.models import ASEMUser
+
+
+class ASEMUserTestCase(TestCase):
+
+    def setUp(self):
+       ASEMUser.objects.create(email="tcamerob@gmail.com",
+        name="Tomas",
+        surname="Camero",
+        birth_date=datetime.datetime(2000, 1, 24, tzinfo=datetime.timezone.utc),
+        sex="Masculino",
+        city="Sevilla",
+        address="Logroño 19",
+        telephone=691644398,
+        postal_code=41730,condition="Esclerosis múltiple",
+        member="Escleriosis múltiple",
+        user_type="SACC",
+        correspondence="Email",
+        status="Casado/a",
+        family_unit_size=4,
+        own_home="Vivienda compartida",
+        own_vehicle=True,
+        bank_account_number="ES6700832134418939683447")
+       
+       
+       ASEMUser.objects.create(email="tcamerob2@gmail.com",
+        name="Tomas2",
+        surname="Camero2",
+        birth_date=datetime.datetime(2000, 1, 24, tzinfo=datetime.timezone.utc),
+        sex="Femenino",
+        city="Sevilla",
+        address="Logroño 20",
+        telephone=691644398,
+        postal_code=41730,
+        condition="ICTUS",
+        member="ELA",
+        user_type="OTROS",
+        correspondence="CC",
+        status="F",
+        family_unit_size=0,
+        own_home="VP",
+        own_vehicle=False,
+        bank_account_number="ES6700832134418939683449")
+       
+    def test_asem_user_create(self):
+        asemuser = ASEMUser.objects.get(name="Tomas")
+        self.assertEqual(asemuser.name, "Tomas")
+        self.assertEqual(asemuser.surname, "Camero")
+        self.assertEqual(asemuser.birth_date.strftime('%Y-%m-%d'), "2000-01-24")
+
+    def test_asem_user_delete(self):
+        asemuser = ASEMUser.objects.get(name="Tomas")
+        asemuser.delete()
```

Comment:
You could verify that the number of users before doing the deletion is equal to 2

Reply:
okey